lib/hammer_cli_foreman_tasks/helper.rb

```diff
@@ -3,10 +3,14 @@ module Helper
 
     # render the progress of the task using polling to the task API
     def task_progress(task_or_id)
-      task_id = task_or_id.is_a?(Hash) ? task_or_id['id'] : task_or_id
-      task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
-      task_progress.render
-      task_progress.success?
+      if !options.values.delete_if(&:empty?).empty?
+        task_id = task_or_id.is_a?(Hash) ? task_or_id['id'] : task_or_id
+        task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
+        task_progress.render
+        task_progress.success?
+      else
+        STDERR.puts "Error: Please mention appropriate attribute value"
```

Comment:
Using `signal_usage_error` helper method following error output is seen, is it the desired way we are expecting :

```ruby
[ERROR 2017-04-06T08:00:54 Exception] Error: Please mention appropriate attribute value

See: 'hammer task progress --help'
Error: Please mention appropriate attribute value

See: 'hammer task progress --help'
[ERROR 2017-04-06T08:00:54 Exception] 

Clamp::UsageError (Please mention appropriate attribute value):
    /home/vagrant/hammer-cli-foreman-tasks/lib/hammer_cli_foreman_tasks/helper.rb:13:in `task_progress'
    /home/vagrant/hammer-cli-foreman-tasks/lib/hammer_cli_foreman_tasks/task.rb:17:in `execute'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/command.rb:63:in `run'
    /home/vagrant/hammer-cli/lib/hammer_cli/abstract.rb:24:in `run'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/command.rb:63:in `run'
    /home/vagrant/hammer-cli/lib/hammer_cli/abstract.rb:24:in `run'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/command.rb:63:in `run'
    /home/vagrant/hammer-cli/lib/hammer_cli/abstract.rb:24:in `run'
    /home/vagrant/.gem/ruby/gems/clamp-1.1.1/lib/clamp/command.rb:132:in `run'
    /home/vagrant/hammer-cli/bin/hammer:125:in `<top (required)>'
    /home/vagrant/.gem/ruby/bin/hammer:23:in `load'
    /home/vagrant/.gem/ruby/bin/hammer:23:in `<top (required)>'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli/exec.rb:74:in `load'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli/exec.rb:74:in `kernel_load'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli/exec.rb:27:in `run'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli.rb:332:in `exec'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli.rb:20:in `dispatch'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/cli.rb:11:in `start'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/exe/bundle:34:in `block in <top (required)>'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/lib/bundler/friendly_errors.rb:100:in `with_friendly_errors'
    /home/vagrant/.gem/ruby/gems/bundler-1.13.6/exe/bundle:26:in `<top (required)>'
    /home/vagrant/bin/bundle:23:in `load'
    /home/vagrant/bin/bundle:23:in `<main>'

```